docs/using_finalizers.md

```diff
@@ -0,0 +1,97 @@
+# Using Finalizers
+
+[Kubernetes operators][operators] in general try to ensure a certain state. The
+create state is already constantly ensured by the create and update events of
+the informer. Deletion events are only executed ones though. In order to
+replicate a continuous reconciliation as well for the delete state a Kubernetes
```

Comment:
Marginal but `also replicate a continuous recon for the` is easier to read for me.